### PR TITLE
Webapp: Improve aligments in the about page 

### PR DESCRIPTION
**Pull request by bastiao**
Improve layout / css in the about page

### CODE REVIEW

File: dicoogle/src/main/resources/webapp/js/components/about/aboutView.js [hunk 1]
```diff
@@ -28,27 +28,30 @@ const AboutView = createReactClass({
...
+            images, without re-engineering or reconfiguration requirements, thus
+            overcoming the limitations of traditional DICOM query services. By
+            presenting the technical assets for plugin development such as a
+            Software Development Kit (SDK), developers are free to expand the
+            archive independently and non-exclusively, without changes to the core
+            platform. This extensible architecture of Dicoogle has enabled its use
+            in research and the healthcare industry, as many use cases can be
+            fulfilled in the same deployment.
+          </div>
           <br />
```
Comment (Enet4) [inline dicoogle/src/main/resources/webapp/js/components/about/aboutView.js:48]:
```suggestion
```

This line break is inelegant.

File: dicoogle/src/main/resources/webapp/sass/components/about.scss [hunk 2]
```diff
@@ -6,9 +6,15 @@
 }
 
 .gridAbout {
-  margin-top: 10px;
+  margin: 10px;
+  padding: 10px;
```
Comment (Enet4) [inline dicoogle/src/main/resources/webapp/sass/components/about.scss:10]: Playing around with these numbers, I got a slightly nicer and tighter look for this grid using the following:

```suggestion
  margin: 0px 10px 10px 0px;
  padding: 5px 10px;
```

**Review by Enet4 — COMMENTED** [review]
Seems OK. I just added a few more suggestions.

Consider also running Prettier on the changed files, as they seem heavily misformatted.

**Comment by bastiao** [comment]
Actions fails due to a non related issue here.

**Review by Enet4 — APPROVED** [review]
:+1:

**Comment by Enet4** [comment]
> Actions fails due to a non related issue here. 

This is under discussion in our internal channel (IEETA).

**Pull request merged**